pkgs/by-name/ri/rime-ice/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "rime-ice";
+  version = "2024.05.21";
```

Comment:
Upstream seems to not cut releases regularly.  I think it is better to use the latest commit together with an update script.

Reply:
What does “ together with an update script” mean？  Is there some mechanism to automatic update packages？

Reply:
I mean adding an update script using passthru.updateScript.

Reply:
I have add passthru.updateScript now, but not sure if it is correct, I have not used it before.